components/authz-service/server/v2/authz.go

```diff
@@ -170,7 +170,8 @@ func (s *authzServer) FilterAuthorizedProjects(
 	}, nil
 }
 
-func isBeta2p1(version api.Version) bool {
+func (s *authzServer) isBeta2p1() bool {
+	version := s.vSwitch.Version
```

Comment:
It's a bit of a hack, but fine, I expected eventually regretting that this field is public 😅 After all, we've already got a "re-architect `authz-service`" card in the backlog.